ocean_lib/utils/addresses_utils.py

```diff
@@ -50,3 +50,24 @@ def get_ocean_address(config):
 
     addresses = get_contracts_addresses(config.address_file, _NETWORK)
     return addresses["Ocean"]
+
+
+def get_staking_address(config):
```

Comment:
These helpers seem to have a lot of duplication. Can we add an argument instead? e.g. `def get_address_of_type(config, address_type)`?